tests/unit/core/api/test_languageTools.js

```diff
@@ -0,0 +1,46 @@
+import * as api from 'core/api';
+import { languageTools } from 'core/api/languageTools';
+import {
+  dispatchClientMetadata,
+} from 'tests/unit/amo/helpers';
+import {
+  createApiResponse,
+  createFakeLanguageAddon,
+} from 'tests/unit/helpers';
+
+
+describe(__filename, () => {
+  let mockApi;
+
+  beforeEach(() => {
+    mockApi = sinon.mock(api);
+  });
+
+  describe('languageTools API', () => {
+    function mockResponse() {
+      return createApiResponse({
+        jsonData: { results: [createFakeLanguageAddon()] },
+      });
+    }
+
+    it('calls the language tools API', async () => {
+      const apiState = dispatchClientMetadata().store.getState().api;
+
+      mockApi
+        .expects('callApi')
+        .withArgs({
+          auth: true,
+          endpoint: 'addons/language-tools',
+          method: 'GET',
+          params: { app: apiState.clientApp },
```

Comment:
Since we don't have Flow coverage in this file, I'm going to suggest something a bit more annoying but safer:

```js
const clientApp = CLIENT_APP_ANDROID;
const apiState = dispatchClientMetadata({ clientApp }).store.getState().api;

mockApi
  ...
  .withArgs({
    ...
    params: { app: clientApp },
  });
```